tests/playwright/Report.md

```diff
@@ -0,0 +1,295 @@
+ const browser = await remote({
+        capabilities: {
+            browserName: 'chrome'
+        }
+  })
+  await browser.url('https://www.wikipedia.org/')
+
+  await browser.getTitle()
+
+  await browser.saveScreenshot('./screenshot.png')
+  await browser.deleteSession();
+})();
+```
+
+### Mocha
+
+The WebdriverIO WDIO runner currently supports Mocha, Jasmine, and Cucumber frameworks and WikiMedia Core test is currently using Mocha framework. [Mocha](https://mochajs.org/) is a feature-rich JavaScript test framework running on Node.js and in the browser, making asynchronous testing simple and fun.
+
+WebdriverIO provides an adapter for Mocha which can easily be added as `@wdio/mocha-framework`.
+
+#### Sample Mocha Code
+
+The code snippet below navigates to wikipedia.org in Chrome and checks if the correct title is being displayed
+
+```js
+const { remote } = require('webdriverio');
+
+describe('Wikipedia home page', async() => {
+  const browser = await remote({
+        capabilities: {
+            browserName: 'chrome'
+        }
+  })
+
+  it('should display correct page title', async() => {
+        await browser.url('https://www.wikipedia.org/')
+        expect(browser).toHaveTitle('The Free Encyclopedia')
+  })
+});
+```
+
+## Playwright
+
+Playwright is a Node.js library to automate Chromium, Firefox, and WebKit with a single API. Playwright is known for the following capability;
+
+- Test across all modern browsers: Single API to automate Chromium, Firefox, and WebKit.
+- Use in your preferred language: Use the Playwright API in JavaScript & TypeScript, Python, C# and, Java.
+- Automate without trade-offs: Capable automation for single-page apps that rely on the modern web platform.
+
+The above points are just a few capabilities. To explore more on Playwright capability, you can check out the Playwright documentation and my [MediaWiki Core fork](https://github.com/hariclerry/mediawiki/tree/master/tests/playwright).
+
+While working with Playwright for the past two months, it has come across as easy to install and setup. The package takes care of installing all the browsers (Chromium, Firefox, and WebKit). Capturing Screenshots is an out-of-the-box experience. However, video recording requires separate installation of [playwright-video and ffmpeg](https://playwright.tech/blog/record-your-browser-tests-with-playwright) but they all blend in with Playwright seamlessly.
+
+Below are some of the benefits I have experienced and seen while using Playwright;
+
+- Console debug option which is very useful in an event when tests are failing with no clear error messages.
+- Scenarios that span multiple pages, domains, and iframes.
+- Auto-wait for elements to be ready before executing actions (like click, fill).
+- Intercept network activity for stubbing and mocking network requests.
+- Seamless integration with Jest.
+
+### Sample Playwright Code
+
+The code snippet below navigates to wikipedia.org in Chrome and saves the screenshot.
```

Comment:
I suggest to make all code samples open a page and check if the title is correct (and not take the screenshot).